.changeset/lazy-hornets-train.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': patch
```

Comment:
We're adding props here which is always a minor version bump. Patch is reserved for bug fixes, minor visual changes etc...

```suggestion
'@shopify/polaris': minor
```